pkg/cli/admin/upgrade/status/health.go

```diff
@@ -21,6 +22,7 @@ const (
 	scopeKindClusterOperator   allowedScopeKind = "ClusterOperator"
 	scopeKindNode              allowedScopeKind = "Node"
 	scopeKindMachineConfigPool allowedScopeKind = "MachineConfigPool"
+	scopeKindAlert             allowedScopeKind = "Alert"
```

Comment:
I do not think we need this. This is used to tie insights to resources in the cluster - either we can do that and we will know the exact kind, or we can and we can have the list of related resources to be empty.